pom.xml

```diff
@@ -30,9 +30,8 @@
         <!-- Build properties -->
         <maven.version>3.3.9</maven.version>
         <java.version>11</java.version>
-        <!-- we use nvm manages the npm version. npm 6 is the default for node 14 -->
         <node.version>v14.17.6</node.version>
-        <npm.version>6.14.15</npm.version>
+        <npm.version>7.23.0</npm.version>
```

Comment:
I remember there was an issue using v7 with node 14. Seems no longer the issue?

Reply:
Thanks for pointing that out, I think the npm version change was a result of jhipster generator. Let me double check